src/Core/src/Handlers/WebView/WebViewHandler.Android.cs

```diff
@@ -79,13 +81,23 @@ public static void MapUserAgent(IWebViewHandler handler, IWebView webView)
 		public static void MapWebViewClient(IWebViewHandler handler, IWebView webView)
 		{
 			if (handler is WebViewHandler platformHandler)
-				handler.PlatformView.SetWebViewClient(platformHandler._webViewClient ??= new MauiWebViewClient(platformHandler));
+				handler.PlatformView.SetWebViewClient(platformHandler._webViewClient ??= platformHandler.GetWebViewClient());
+		}
+
+		protected virtual WebViewClient GetWebViewClient()
```

Comment:
This is the main change.

Reply:
Yeah, it would be possible, although these changes would allow to override the WebViewClient in the same way that it was done in Xamarin.Forms: https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/Renderers/WebViewRenderer.cs#L102

Reply:
Problem with this is if i now replace the Mapper, it will not call the protected method.

Reply:
Should we close the PR?